nilearn/datasets/tests/test_neurovault.py

```diff
@@ -284,7 +284,7 @@ def test_get_batch(tmp_path):
 
     assert "results" in batch
     assert "count" in batch
-    with open(tmp_path / "test_nv.txt", "w"):
+    with (tmp_path / "test_nv.txt").open("w"):
         pass
```

Comment:
what does this do?
Is this even needed: does the test fail if you remove it?

if still needed I think we can just have

```suggestion
    (tmp_path / "test_nv.txt").touch()
```

Reply:
Let's check that in another issue / PR

Reply:
@Remi-Gau good catch, indeed it does nothing. Will open an issue/PR to remove it